src/OrchardCore.Themes/TheAdmin/Assets/scss/modules/admin/_codemirror.scss

```diff
@@ -5,6 +5,7 @@
 
 .CodeMirror-fullscreen {
     z-index: 1000;
+    top: 0px !important;
```

Comment:
@Skrypt an improvement could be made by moving `TheAdmin.css` first, so it could override the default styles. I'm wondering how the `z-index` for instance is applied here?

Reply:
I could start a new PR to move it above, but I'm not sure if there`s an implications that you know and why it's moved down